### PR TITLE
Eq 785 section numbers

### PR DESCRIPTION
**Pull request by bennpearson**
### What is the context of this PR?
Section and question numbers 
 
### How to review 
Choose 1_0001.json schema and check section and question numbers

### CODE REVIEW

**Comment by collisdigital** [comment]
@bennpearson please attach screenshots of Census and Business themes section and question numbers both left-hanging and in-line... thanks

File: app/assets/styles/partials/components/_question.scss [hunk 1]
```diff
@@ -38,6 +39,14 @@
   }
 }
 
+.question__title__number {
```
Comment (hamishtaplin) [inline app/assets/styles/partials/components/_question.scss]: Double BEM 👎

File: app/assets/styles/partials/objects/_section.scss [hunk 2]
```diff
@@ -15,6 +15,62 @@
...
+}
+
+.section__title__number {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: $color-white;
+  position: relative;
+  text-align: center;
+  min-width: 48px;
```
Comment (hamishtaplin) [inline app/assets/styles/partials/objects/_section.scss]: Any reason not to use `rem` here?

File: app/assets/styles/partials/objects/_section.scss [hunk 3]
```diff
@@ -15,6 +15,62 @@
   }
 }
 
+.section__title {
+  display: flex;
+  align-items: center;
+}
+
+.section__title__number {
```
Comment (hamishtaplin) [inline app/assets/styles/partials/objects/_section.scss]: Double BEM 👎

**Review by hamishtaplin — CHANGES_REQUESTED** [review]

**Comment by hamishtaplin** [comment]
screenshot 2017-02-01 13 36 34

Section numbers a bit big on mobile maybe?

File: app/assets/styles/partials/objects/_section.scss [hunk 4]
```diff
@@ -15,6 +15,62 @@
...
+    height: auto;
+    top: 0;
+    position: relative;
+    font-size: 1.55556rem;
+  }
+  &::after {
+    content: "";
+    background-color: $color-section-number;
+    border-radius: 100%;
+    width: 48px;
```
Comment (hamishtaplin) [inline app/assets/styles/partials/objects/_section.scss]: Don't use magic numbers, use a var for this as it's repeated throughout.

File: app/assets/styles/partials/objects/_section.scss [hunk 5]
```diff
@@ -15,6 +15,62 @@
...
+    height: 48px;
+    position: absolute;
+    display: block;
+    z-index: -1;
```
Comment (hamishtaplin) [inline app/assets/styles/partials/objects/_section.scss]: Risk of this ending up behind something if you use `-1`?

**Review by hamishtaplin — CHANGES_REQUESTED** [review]

**Comment by hamishtaplin** [comment]
I think there's an issue with using fixed values for positioning as it starts to look weird in some places: 

screenshot 2017-02-01 13 43 27

**Comment by collisdigital** [comment]
@hamishtaplin did a visual review with @alexmorris and @bennpearson at desk just now, some updates coming, including spacing on mobile, watch this space.

File: app/assets/styles/partials/components/_question.scss [hunk 6]
```diff
@@ -38,6 +39,14 @@
...
+  padding-right: 0.5rem;
+  @include mq(xl) {
+    position: absolute;
+    left: -48px;
```
Comment (hamishtaplin) [inline app/assets/styles/partials/components/_question.scss]: If you use `text-align: right` and then give this a fixed width then it'll scale better with bigger numbers:

screenshot 2017-02-01 14 08 16

screenshot 2017-02-01 14 08 37

**Review by hamishtaplin — CHANGES_REQUESTED** [review]

File: app/assets/styles/partials/objects/_section.scss [hunk 7]
```diff
@@ -15,6 +15,61 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: $color-white;
+  position: relative;
+  text-align: center;
+  min-width: 48px;
+  height: 48px;
+  z-index: 1;
+  top: 1px;
```
Comment (hamishtaplin) [inline app/assets/styles/partials/objects/_section.scss]: Some issue with positioning somewhere around here when errors exist in page

screenshot 2017-02-01 14 59 41

**Review by hamishtaplin — CHANGES_REQUESTED** [review]

**Review by hamishtaplin — APPROVED** [review]

**Pull request merged**